.appveyor.yml

```diff
@@ -0,0 +1,25 @@
+skip_commits:
```

Comment:
what is this file about? Is it ok that uses windows?

Reply:
@kuzeko I have no idea, this file was part of both the arborist script and the skeleton app, and it seemed nothing had to be changed.

Reply:
Interesting, let's assume it works then